ui/perfherder/alerts/AlertTable.jsx

```diff
@@ -248,6 +248,17 @@ export default class AlertTable extends React.Component {
                   </tr>
                 </thead>
                 <tbody>
+                  <tr>
+                    <th>&nbsp;</th>
```

Comment:
In react, you can also do {' '} for an empty space. But can't they just be empty without a space?